### DISCUSSION TITLE
Do you or anyone you know suffer from tinnitus?

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#982] [2023 Attraction Contributions]
Do you or anyone you know suffer from tinnitus?  What is your experience, discuss with others here!

**Comment by hernessteve** [comment]
I have audio hallcinations. Music, talking, mine is usually Big Band music. I also have slamming noises and people speaking or yelling directly next to me, as well as caling my name as I fall asleep. Its during the sleep phase that is used for hypnosis.  I have had all of this for decades. I have a rare neurological disorder called Ideopathic Hypersomnia, a sleep disorder that loves to throw everything but the kitchen sink into the game. I also have chronic migraine disease, seems to run in my family.  I only developed Tinnitus a couple of years ago as a side effect of Bupropion, aka Wellbutrin SR. This, like the seizure medications, and the Narcolepsy meds, is often used for off label treatment of various conditions, including neuro pain. Stopping the Buprophion did not stop the Tinnitus, unfortunately. I am at the point where I need to have my hearing checked, but I wonder if it is actually my hearing that is going, or just that the noise in my brain, not my ears, is simply too loud to let me hear past it.

**Comment by MirkosMangle** [comment]
I have excellent hearing, in fact, I'm a musician. I've also had tinnitus for as long as I can remember, so this explanation of the cause being due to inner ear damage can't be correct for everyone. Also, it's never bothered me; if anything, I find it rather interesting. I have no idea what total silence "sounds" like. In fact, the quieter the environment is, the louder my tinnitus is, probably because the tinnitus isn't being drowned out by other sounds. I already started to research [hearing aids online](https://audiologyisland.com/online-shop/) and not denying its gonna happen to me one day.
